src/components/vectorfeedback/partials/vectorfeedback.html

```diff
@@ -0,0 +1,87 @@
+<div id="ga-feedback-vector" class="ga-vector-feedback-panel col-xs-12 col-sm-6 col-md-5 col-lg-5 collapse in"
+    aria-expanded="true">
+    <div class="ga-vector-feedback-header">
+        <div class="ga-vector-feedback-title pull-left">Vector Tiles Alpha Viewer</div>
+        <div class="ga-vector-feedback-btn-wrapper">
+            <button class="pull-right ga-icon ga-btn fa fa-remove-sign"
+                    data-toggle="collapse"
+                    data-target="#ga-feedback-vector"
+                    aria-expanded="true">
+            </button>
+        </div>
+    </div>
+    <p>
+        Lorem ipsum dolor sit amet, nam an veritus adipiscing, mutat habemus sea an. Ne decore nemore scaevola mel,
+        vim at qualisque consulatu. Pro ponderum molestiae dignissim et. Est id ferri meliore atomorum, in mei
+        primindoctum. Ea sint dolore nec, ea dicam propriae verterem est.
+    </p>
+    <hr>
+    <div class="ga-vector-feedback-item">
+        <h4 class="pull-left">Select a layer</h4>
+        <div class="ga-vector-feedback-item-select-wrapper">
+            <select class="form-control"
+                    ng-model="options.backgroundLayer"
+                    ng-options="c.label for c in options.backgroundLayers track by c.id">
+            </select>
+        </div>
+    </div>
+    <div class="ga-vector-feedback-item">
+        <h4 class="pull-left">Select items in the map</h4>
+        <div class="ga-vector-feedback-item-select-wrapper">
+            <select class="form-control"
+                    ng-model="options.selectedLayer"
+                    ng-options="c.label for c in options.layers[options.backgroundLayer.id].selectableLayers track by c.value">
+            </select>
+        </div>
+    </div>
+    <div class="ga-vector-feedback-item">
+        <h4 class="pull-left">Change selected item color</h4>
+        <div class="ga-vector-feedback-item-select-wrapper">
+            <select class="form-control ga-vector-feedback-colors"
+                    ng-model="options.color"
+                    ng-options="c.label for c in options.colors track by c.value">
```

Comment:
You can use a `ng-repeat` on a `<option>` elt  to have the possibility to specify a css style with the color value